Sources/Subs.php

```diff
@@ -3438,7 +3448,7 @@ function custMinify($data, $type, $do_deferred = false)
 		{
 			loadLanguage('Errors');
 			log_error(sprintf($txt['file_not_created'], $toCreate), 'general');
-			continue;
+			return false;
```

Comment:
How will this  behave with multiple themes?

Lets say there are 3 active themes and the second one fails, with a return the third one will never be processed.